.travis.yml

```diff
@@ -13,6 +13,12 @@ matrix:
     - os: osx
       osx_image: xcode11.3
       env: PDFium_BRANCH=$TRAVIS_BRANCH PDFium_V8=enabled CONFIGURATION=Release
+    - os: osx
+      osx_image: xcode12.2
+      env: PDFium_BRANCH=$TRAVIS_BRANCH PDFium_V8=disabled CONFIGURATION=Release TARGET_CPU=arm64
+    - os: osx
```

Comment:
Since the V8 build takes more than 2 hours, I suggest we remove it, as I did for the Windows version